client/state/purchases/test/reducer.js

```diff
@@ -142,18 +142,11 @@ describe( 'items', () => {
 			purchaseId: 2
 		} );
 
-		expect( getByPurchaseId( state, 2 ) ).to.be.eql( {
-			data: {
-				id: 2,
-				error: 'Unable to fetch stored cards',
-				siteId,
-				userId
-			},
-			error: null,
-			isFetchingSitePurchases: false,
-			isFetchingUserPurchases: false,
-			hasLoadedSitePurchasesFromServer: true,
-			hasLoadedUserPurchasesFromServer: true
+		expect( getByPurchaseId( { purchases: state }, 2 ) ).to.be.eql( {
```

Comment:
I probably should check the full context, this is the limitation of GH diffs ;)
We probably can leave it as it is in that case.